CHANGES.txt

```diff
@@ -15,6 +15,7 @@
  * Add new CQL function maxWritetime (CASSANDRA-17425)
  * Add guardrail for ALTER TABLE ADD / DROP / REMOVE column operations (CASSANDRA-17495)
  * Rename DisableFlag class to EnableFlag on guardrails (CASSANDRA-17544)
+ * `nodetool version` now includes GitSHA
```

Comment:
CHANGES list is reverse chronological (you put new entries in at the top, not the bottom)